app/javascript/dashboard/helper/messageHelper.js

```diff
@@ -0,0 +1,63 @@
+const MESSAGE_VARIABLES_REGEX = /{{(.*?)}}/g;
+export const replaceVariablesInMessage = ({ message, variables }) => {
+  return message.replace(MESSAGE_VARIABLES_REGEX, (match, replace) => {
+    return variables[replace.trim()]
+      ? variables[replace.trim().toLowerCase()]
+      : '';
+  });
+};
+
+const skipCodeBlocks = str => str.replace(/```(?:.|\n)+?```/g, '');
+
+export const getFirstName = ({ name }) => {
+  return name.split(' ')[0];
+};
+
+export const getLastName = ({ name }) => {
+  return name.split(' ').length > 1
+    ? name.split(' ')[name.split(' ').length - 1]
+    : '';
+};
+
+export const getMessageVariables = ({ conversation }) => {
+  const {
+    meta: { assignee = {}, sender = {} },
+    id,
+  } = conversation;
+
+  return {
+    'contact.name': sender?.name,
+    'contact.first_name': getFirstName({ name: sender?.name }),
+    'contact.last_name': getLastName({ name: sender?.name }),
+    'contact.email': sender?.email,
+    'contact.phone': sender?.phone_number,
+    'conversation.id': id,
+    'agent.name': assignee?.name ? assignee?.name : '',
+    'agent.first_name': assignee?.name
+      ? getFirstName({ name: assignee?.name })
+      : '',
+    'agent.last_name': assignee?.name
+      ? getLastName({ name: assignee?.name })
+      : '',
+    'agent.email': assignee?.email ? assignee?.email : '',
```

Comment:
```suggestion
    'agent.email': assignee?.email ?? '',
```

Using the logical nullish operator, much simpler to read